fsspec/fuse.py

```diff
@@ -1,32 +1,44 @@
+from __future__ import print_function
+
+import argparse
 import logging
 import os
+import pathlib
 import stat
 import threading
 import time
 from errno import EIO, ENOENT
 
-from fuse import FUSE, FuseOSError, Operations
+from fuse import FUSE, FuseOSError, LoggingMixIn, Operations
+
+from fsspec._version import get_versions
```

Comment:
You should be able to replace this with `fsspec.__version__`.

Reply:
`fsspec.__version__` is way better, will use it.